content/workload-reduction-toolkit/identify-workload-issues/act-on-your-staff-workload-survey.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Act on your staff workload survey
+colour: pink
+---
+
+# Act on your staff workload survey
+
+<strong class="govuk-tag">Presentation</strong>
+
+This presentation will help you to support a discussion of the results of your
+staff workload survey, prioritise next steps and identify:
+
+- how you can reduce workload without affecting pupil outcomes
+- what you can streamline whilst maintaining standards
+- any areas that are not covered in the survey results
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/download-icon.svg" alt="Download icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Download the presentation
+      </h2>
+      <div class="govuk-grid-row info-box__download-content">
+        <div class="govuk-grid-column-one-half">
+          <img src="/assets/images/preview-placeholder.jpg" alt="Placeholder image" class="dfe-file-preview-image">
+        </div>
+        <div class="govuk-grid-column-one-half">
+          <p class="govuk-body-s">
+            <strong>Presentation length:</strong> 40 minutes
+          </p>
+          <p class="govuk-body-s">
+            <strong>Resource type:</strong> Presentation
+          </p>
+          <p class="govuk-body-s">
+            <strong>File type:</strong> TBC
+          </p>
+          <p>
+            <a class="govuk-link govuk-link--no-visited-state" href="<%= @base_url %>/assets/files/TBC">
+              Download
+            </a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+## Audience
+
+This resource has been provided to support senior leadership teams to work with
+the whole school, teams or departments.
+
+## Presentation length
```

Comment:
Content question - No other presentation resource repeats the presentation length in it's own paragraph - it's already stated in the box above too. Do we want to keep it?

Reply:
I'm a bit confused so might be worth a Slack call, as I can see it on all the presentations uploaded so far?

https://improve-workload-and-wellbeing-for-school-staff.education.gov.uk/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications/

https://improve-workload-and-wellbeing-for-school-staff.education.gov.uk/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/review-and-streamline-lesson-planning/

Reply:
apologies - my confusion! i must have searched within a subset of files or something by mistake!